packages/cfd/src/Helpers/cfd-config.ts

```diff
@@ -59,6 +59,7 @@ export const MARKET_TYPE_SHORTCODE = {
     FINANCIAL_LABUAN: 'financial_stp_labuan',
     FINANCIAL_MALTA_INVEST: 'financial_maltainvest',
     FINANCIAL_GOLD: 'financial_gold',
+    CFDS_GOLD: 'CFDs_gold',
```

Comment:
isn't it MT5_GOLD?

Reply:
The `MarketType` is set to `CFDs` for the EU. I'm concatenating `MarketType` with the `product` to create a shortcode, which is how it becomes `CFDs_gold`

<img width="735" alt="Screenshot 2024-12-06 at 6 02 07 PM" src="https://github.com/user-attachments/assets/60eb4935-006a-4582-a7a8-b61c394e843e">